src/wmtk/Mesh.hpp

```diff
@@ -24,6 +24,31 @@ class Mesh
     friend class ParaviewWriter;
     friend class MeshReader;
 
+
+    // Interface for SyncMesh
+
+    // TODO: Need a wrapper for MeshAttributeHandle<Tuple>?
+
+    // size of dim(Mesh), store base_tuple for each simplex
+    std::vector<MeshAttributeHandle<long>> base_tuple_handle;
```

Comment:
We need to decide on a name for these tuples, I'm not happy with base, perhaps sticking it in a container would help isolate a name like handle as this is a attribute for a collection of handles?